src/style.css

```diff
@@ -1 +1,32 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 22px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  height: 116px;
+  width: 96px;
+}
+
+.stars {
```

Comment:
According to the task conditions, each BEM block should have its separate CSS file. In your solution all the styles are placed in the single file.